core/src/main/scala/kafka/admin/AdminClient.scala

```diff
@@ -165,14 +174,30 @@ class AdminClient(val time: Time,
                                   consumers: Option[List[ConsumerSummary]],
                                   coordinator: Node)
 
-  def describeConsumerGroup(groupId: String): ConsumerGroupSummary = {
-    val coordinator = findCoordinator(groupId)
+  def describeConsumerGroupHandler(coordinator: Node, groupId: String): GroupMetadata = {
     val responseBody = send(coordinator, ApiKeys.DESCRIBE_GROUPS,
         new DescribeGroupsRequest.Builder(Collections.singletonList(groupId)))
     val response = responseBody.asInstanceOf[DescribeGroupsResponse]
     val metadata = response.groups.get(groupId)
     if (metadata == null)
       throw new KafkaException(s"Response from broker contained no metadata for group $groupId")
+    metadata
+  }
+
+  def describeConsumerGroup(groupId: String): ConsumerGroupSummary = describeConsumerGroup(groupId, 0)
```

Comment:
I just realized why I kept this. There are a couple of places in the Java files ([here](https://github.com/apache/kafka/blob/2e662a061608a0755274da3e2ccf87401936febf/core/src/main/scala/kafka/tools/StreamsResetter.java#L94) and [here](https://github.com/apache/kafka/blob/trunk/streams/src/test/java/org/apache/kafka/streams/integration/ResetIntegrationTest.java#L425)) that this method is called. It seems that if the method is called in Java files the default parameter value is not recognized and I get compilation error. Not sure if there is a workaround, as I haven't found one. So I'll keep it for now.